data-prepper-plugins/aws-sqs-common/src/main/java/org/opensearch/dataprepper/plugins/aws/sqs/common/model/SqsOptions.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.aws.sqs.common.model;
+
+import java.time.Duration;
+
+public class SqsOptions {
+    private static final int DEFAULT_MAXIMUM_MESSAGES = 10;
```

Comment:
Let's not use a default here. Use `null` which will use the SQS queue defaults.

Reply:
Thanks @dlvenable for your review, Sure i will make the changes accordingly.

Reply:
@kkondaka we have incoporated the david suggested changes. Please review.